rfcs/0008-commitstatus/README.md

```diff
@@ -0,0 +1,71 @@
+
+# RFC-xxxx Create a new API kind for Commit Status
+
+**Status:** provisional
+
+**Creation date:** 2022-05-12
+
+**Last update:** 2022-05-12
+
+## Summary
+
+There should be a dedicated kind in the notification controller for sending commit status notifications to git providers.
```

Comment:
I would expect the summary to be written as `[This RFC] <summary>`. So in this particular case, something like:

> This RFC proposes to create a dedicated `CommitStatus` kind in the <notification controller API group> to allow configuration of commit status notifications to a (Git) `Provider`.